multitenant/form_handler.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+require_once('../config/settings.php');
+session_start();
+
+/**
+ * Verify that .htaccess and .htpasswd files exist. Very basic security check, but it offers a reminder to the end user.
+ *
+ * @return bool
+ */
+function multitenant_check_security()
+{
+    $secure = false;
+
+    if (file_exists('.htaccess')) {
```

Comment:
Using .htaccess and .htpasswd will definitely work for now, but I don't feel like it's a permanent solution.  Two other options would be to use Composer commands or to create a full-fledged login system.  Thoughts?

Reply:
Agreed, but it feels like adding a full-fledged login system is overkill for managing a single table. Maybe Composer is a better option, but I have no experience in doing that.
Cidi Labs is actually using a separate system, Drupal, that will write to the institutes table in the customer setup process.

Reply:
Ahh, ok.  I didn't realize you were using a separate system.  I hate to lose the work you put into the admin interface, but I think it would be better to remove it from this pull request.  If an institution wants to do multitenant right now, they can add the institutions to the DB manually.  I will put in an issue for creating the Composer command to add institutions.